src/commands/force/community/create.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { JsonMap } from '@salesforce/ts-types';
+import { Messages } from '@salesforce/core';
+import { CommunityNameValueParser } from '../../../lib/community/commands/CommunityNameValueParser';
+import { ConnectExecutor } from '../../../lib/connect/services/ConnectExecutor';
+import { CommunityCreateResource } from '../../../lib/community/connect/CommunityCreateResource';
+import { CommunityCreateResponse } from '../../../lib/community/defs/CommunityCreateResponse';
+
+Messages.importMessagesDirectory(__dirname);
+const communityMessages = Messages.loadMessages('salesforce-alm', 'community_commands');
```

Comment:
this should be loading messages from this package, `loadMessages('@salesforce/plugin-community'...)`

Reply:
Yep, got this updated 👍